pandas/core/arrays/timedeltas.py

```diff
@@ -651,8 +651,7 @@ def __floordiv__(self, other):
 
             # at this point we should only have numeric scalars; anything
             #  else will raise
-            result = self.asi8 // other
-            np.putmask(result, self._isnan, iNaT)
+            result = self._ndarray / other
```

Comment:
why is this using div instead of floordiv?

Reply:
Because I was stupid .. :), and because the test I added (or any other test we already have) actually don't catch this because only use such data that it is the same for div or floordiv ..

Reply:
-> https://github.com/pandas-dev/pandas/pull/44471